src/GovUk.Education.ExploreEducationStatistics.Publisher/Functions/StageScheduledReleasesFunction.cs

```diff
@@ -1,10 +1,18 @@
-﻿using System;
+﻿#nullable enable
+
```

Comment:
Just wondering if this space is needed?

Reply:
Nope, and it looks like our general convention is to not have one, so removed it ta